hazelcast/src/test/java/com/hazelcast/version/VersionTest.java

```diff
@@ -40,6 +56,14 @@ public void isGreaterThan() throws Exception {
     }
 
     @Test
+    public void isUnknownOrGreaterThan() throws Exception {
```

Comment:
`Version` models cluster version which is `major.minor`, no patch level information is maintained.